impl/src/main/java/org/ehcache/internal/store/DefaultStoreProvider.java

```diff
@@ -40,6 +41,7 @@
 /**
  * @author Ludovic Orban
  */
+@SupplementaryService
```

Comment:
How will the `CacheManager` distinguish between the transactional store provider and the default one if both are available?